src/IO.Milvus/Client/AbstractMilvusGrpcClient.cs

```diff
@@ -1,8 +1,6 @@
-﻿using Google.Protobuf;
-using Grpc.Core;
+﻿using Grpc.Core;
 using Grpc.Net.Client;
 using IO.Milvus.Exception;
-using IO.Milvus.Grpc;
```

Comment:
Previously, I have been using the IO.Milvus.Grpc namespace, which has been added to the Milvus-Proto repository in this [pr](https://github.com/milvus-io/milvus-proto/pull/56). However, upon checking the [v2.2](https://github.com/milvus-io/milvus-proto/blob/2.2/proto/common.proto) code in milvus-proto, It seems that the 2.2 and master branches have not been merged yet.

Reply:
@yhmo It seems that there have been no changes merged from the master branch to [branch 2.2](https://github.com/milvus-io/milvus-proto/network) since October 14, 2022. However, a PR for the C# namespace was merged into the master branch. Should I create a PR to merge it into 2.2?

Reply:
Ok, I see. Let me cherry-pick the pr to milvus-proto 2.2 branch.
For milvus, the master branch and 2.2 branch might use different proto(a bit difference). So, we should create a 2.2 branch for the csharp sdk repo, just like the other sdks(java/go/python sdk).

Reply:
I have cherry-picked the pr to milvus-proto 2.2 branch: https://github.com/milvus-io/milvus-proto/pull/146